frontendservice/src/App.vue

```diff
@@ -16,11 +16,12 @@ specific language governing permissions and limitations under the License. */
 import firebase from "firebase";
 
 export default {
-  name: "App",
+  name: "App"
 };
 </script>
 
 <style>
+@import "~bulma/css/bulma.css";
```

Comment:
This is not required I guess because I have already included the Bulma stylesheet link in index.html. Ishita and I are working using that, please check this.